### DISCUSSION TITLE
Can you optimize the display style of the content of `front matter` in `markdown`?

### DISCUSSION THREAD

**Discussion by misitebao** [community/community#8732] [Other Feature Feedback, Questions, & Ideas]
Because most of the `markdown` files in the repository have `front matter` (including json, yaml, toml, etc.), but because it is not a standard markdown grammar, it cannot be parsed, causing it to be in plain text The method is displayed at the top of the document, which looks ugly. Can github perform additional analysis of these content and optimize the display style?

Screenshot:
![image](https://user-images.githubusercontent.com/28185258/145701260-348ae9c9-31df-431e-9897-dd715ecfd49e.png)

**Comment by bewuethr** [comment]
If you delimit the front matter with `---` instead of `+++`, it gets rendered as a table; example:

```markdown
---
title: Council of the Swiss Abroad 2017
author: Ernst Notz
last_modified_at: 2022-05-23
tag: aso
---
```

becomes

![image](https://user-images.githubusercontent.com/8521043/200956561-0995b4e3-a3f2-4967-b861-bed58d34e229.png)

**Reply by misitebao** [reply]
I want to use TOML instead of YAML.
